linux/keyman-config/keyman_config/ibus_util.py

```diff
@@ -114,10 +114,8 @@ def verify_ibus_daemon(start):
         return IbusDaemon.ERROR_USER
 
     try:
-        ps_output = subprocess.run(('ps', '--user', user, '-o', 's=', '-o', 'cmd'),
-                                   check=False, stdout=subprocess.PIPE).stdout
-        ibus_daemons = re.findall(r'^[^ZT] (/[^ ]+/|)ibus-daemon( .*|$)',
-                                  ps_output.decode('utf-8'), re.MULTILINE)
+        ibus_daemons = subprocess.run(('pgrep', '-u', user, 'ibus-daemon'),
+                                      check=False, stdout=subprocess.PIPE).stdout.split()
```

Comment:
Assuming pgrep is always available on all systems? It's a lot cleaner so I like it otherwise!